app/backend/prepdocslib/searchmanager.py

```diff
@@ -70,92 +72,95 @@ async def create_index(self, vectorizers: Optional[List[VectorSearchVectorizer]]
         logger.info("Checking whether search index %s exists...", self.search_info.index_name)
 
         async with self.search_info.create_search_index_client() as search_index_client:
-
-            if self.search_info.index_name not in [name async for name in search_index_client.list_index_names()]:
-                logger.info("Creating new search index %s", self.search_info.index_name)
-                fields = [
-                    (
-                        SimpleField(name="id", type="Edm.String", key=True)
-                        if not self.use_int_vectorization
-                        else SearchField(
-                            name="id",
-                            type="Edm.String",
-                            key=True,
-                            sortable=True,
-                            filterable=True,
-                            facetable=True,
-                            analyzer_name="keyword",
-                        )
-                    ),
-                    SearchableField(
-                        name="content",
+            fields = [
```

Comment:
How necessary is it to store md5 as a field in the search index? I was thinking we'd also need to store it in the blob/file metadata? @mattgotteiner What do you think of storing md5 in search index, have you seen this approach before?